kubernetes-model/kubernetes-model/pom.xml

```diff
@@ -129,6 +129,8 @@
                 <delete file="${project.build.directory}/generated-sources/io/fabric8/kubernetes/api/model/Quantity.java" verbose="true" />
                 <delete file="${project.build.directory}/generated-sources/io/fabric8/kubernetes/api/model/WatchEvent.java" verbose="true" />
                 <delete file="${project.build.directory}/generated-sources/io/fabric8/openshift/api/model/Template.java" verbose="true" />
+                <delete file="${project.build.directory}/generated-sources/io/fabric8/kubernetes/api/model/admission/AdmissionRequest.java" verbose="true" />
+                <delete file="${project.build.directory}/generated-sources/annotations/io/fabric8/kubernetes/api/model/admission/AdmissionRequestFluent.java" verbose="true" />
```

Comment:
File removed from code generation, but not added instead, is this OK?

Reply:
Actually, I was facing problems during compilation with builder classes generated by Sundrio for `AdmissionRequest`. I had to remove them in order to compile successfully. But unfortunately, builders won't be available corresponding to this class due to this change.

Reply:
Can't we just add AdmissionRequestFluent same as AdmissionRequest?

Reply:
okay, let me try adding those